test/leak/async_hooks.js

```diff
@@ -0,0 +1,109 @@
+'use strict'
+
+const tracer = require('../../')
+
+const SCOPE = require('../../src/scope/new/scope')
+
+tracer.init({
+  scope: SCOPE
+})
```

Comment:
My guess is the intermittent leak failure is from the zipkin writer.  Other plugins set `sampleRate: 0` to avoid bringing it into the mix.